internal/pkg/mqtt/client.go

```diff
@@ -231,6 +231,8 @@ func newMessageHandler(
 			errorChannel <- err
 		}
 
+		messageEnvelope.ReceivedTopic = message.Topic()
```

Comment:
message.Topic() is formatted more like a URI, so it can even contain optional parameters. So it would have been nice to call this messageEnvelope.ReceivedUri, which may allow reusing the same member for HTTP based messages as well.

Reply:
Had thought about how/if the HTTP Trigger might use this, but since the URL is fixed, i.e. /api/v2/trigger, don't have a need for it to.